.github/workflows/ci.yml

```diff
@@ -16,13 +16,14 @@ jobs:
         include:
           - elixir: 1.11.4
             otp: 22.3
-            warnings-as-errors: true
           - elixir: 1.11.4
             otp: 23.3
-            warnings-as-errors: true
           - elixir: 1.12.1
             otp: 24.0
             check_formatted: true
+          - elixir: 1.13.0-rc.1
+            otp: 24.0
+            run_plugin_tests: true
             warnings-as-errors: true
```

Comment:
@msaraiva The plugin test will fail before Elixir 1.13 because it's dependent upon the Mix Formatter Plugin feature. So here, I'm adding Elixir 1.13(-rc.1) to the CI matrix and only allowing that test to run for that version.

It turns out that the following two tests fail on Elixir 1.13. This happens on `master` as well; it's unrelated to this PR.

```
  1) test nested if/elseif/else parser error message contains the correct line with nested struct (Surface.Constructs.IfTest)
Error:      test/surface/constructs/if_test.exs:321
     Expected exception Surface.Compiler.ParseError but got CompileError (code: invalid module name: :"Elixir.Surface.Constructs.IfTest.(test nested if/elseif/else parser error message contains the correct line with nested struct) at line 349")
     code: assert_raise(Surface.Compiler.ParseError, message, fn ->
     stacktrace:
       (elixir 1.13.0-rc.1) src/elixir_module.erl:97: :elixir_module.compile/5
Warning:        (elixir 1.13.0-rc.1) src/elixir_lexical.erl:15: :elixir_lexical.run/3
       (stdlib 3.16.1) erl_eval.erl:685: :erl_eval.do_apply/6
Warning:        (elixir 1.13.0-rc.1) src/elixir.erl:289: :elixir.recur_eval/3
       (elixir 1.13.0-rc.1) src/elixir.erl:274: :elixir.eval_forms/3
Warning:        (elixir 1.13.0-rc.1) lib/code.ex:797: Code.eval_quoted/3
       test/surface/constructs/if_test.exs:349: anonymous fn/0 in Surface.Constructs.IfTest."test nested if/elseif/else parser error message contains the correct line with nested struct"/1
Warning:        test/surface/constructs/if_test.exs:348: (test)


  2) test nested if/elseif/else compile error message contains the correct line with nested struct (Surface.Constructs.IfTest)
Error:      test/surface/constructs/if_test.exs:290
     Wrong message for CompileError
     expected:
       "code:12: invalid value for property \"prop\". Expected a :list, got: \"some string\"."
Warning:      actual:
       "code: invalid module name: :\"Elixir.Surface.Constructs.IfTest.(test nested if/elseif/else compile error message contains the correct line with nested struct) at line 317\""
     code: assert_raise(CompileError, message, fn ->
     stacktrace:
       test/surface/constructs/if_test.exs:316: (test)
```